src/modules/components/proposal/proposalVoting/proposalVotingDetails/proposalVotingDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import { DateFormat, DefinitionList, formatterUtils, Heading, type ITabsContentProps, Tabs } from '../../../../../core';
+import { useOdsModulesContext } from '../../../odsModulesProvider';
+import { ProposalVotingTab } from '../proposalVotingDefinitions';
+import { useProposalVotingStageContext } from '../proposalVotingStageContext';
+
+export interface IProposalVotingDetailsSetting {
+    /**
+     * Term of the setting.
+     */
+    term: string;
+    /**
+     * Value of the setting.
+     */
+    value: string;
+}
```

Comment:
The types on the proposal-voting component won't grow, we'll only have these. On the ProposalActions instead we'll have one more type for each action we'll add/support. I see the `ProposalActions` case as special as it will have many types